generator/java/resources/templates/StatisticsApplicationExample.mustache

```diff
@@ -1,11 +1,11 @@
 package com.criteo.marketing.examples;
 
-import com.criteo.marketing.ApiClient;
-import com.criteo.marketing.ApiException;
-import com.criteo.marketing.ApiResponse;
-import com.criteo.marketing.Configuration;
-import com.criteo.marketing.api.AnalyticsApi;
-import com.criteo.marketing.model.StatisticsReportQueryMessage;
+import {{invokerPackage}}.ApiClient;
+import {{invokerPackage}}.ApiException;
+import {{invokerPackage}}.ApiResponse;
+import {{invokerPackage}}.Configuration;
+import {{apiPackage}}.AnalyticsApi;
+import {{modelPackage}}.StatisticsReportQueryMessage;
 import java.time.OffsetDateTime;
 import java.util.Arrays;
 import java.util.List;
```

Comment:
Line 25, 26: Dates should be updated `OffsetDateTime.parse("2022-07-01T00:00:00+01:00", DateTimeFormatter.ISO_OFFSET_DATE_TIME)`